helix-term/src/ui/completion.rs

```diff
@@ -15,19 +15,28 @@ use helix_lsp::{lsp, util};
 use lsp::CompletionItem;
 
 impl menu::Item for CompletionItem {
-    fn sort_text(&self) -> &str {
-        self.filter_text.as_ref().unwrap_or(&self.label).as_str()
+    type EditorData = ();
+    fn sort_text(&self, _data: &Self::EditorData) -> Cow<str> {
+        self.filter_text
+            .as_ref()
+            .unwrap_or(&self.label)
+            .as_str()
+            .into()
     }
```

Comment:
Could call the `filter_text` method and mark as inline so there's no code duplication